classes/testing_idealist.rb

```diff
@@ -61,12 +61,12 @@ def estimate_testingness_in_file file_path
       return has_no_tests
     end
     
-    regexes = [/XCTAssert|XCTFail/,        # XCTest
-               /expect\(/,                 # Expecta, Nimble
-               /should\]|shouldNot\]/,     # Kiwi
-               /assertThat/,               # OCHamcrest
-               / should .*;| should_not /, # Cedar
-               /FBSnapshotVerify/          # FBSnapshotTestCase
+    regexes = [/XCTAssert|XCTFail/,                 # XCTest
+               /expect\(/,                          # Expecta, Nimble
+               /should\]|shouldNot\]/,              # Kiwi
+               /assertThat/,                        # OCHamcrest
+               / should .*;| should_not |expect\(/, # Cedar
```

Comment:
this should already be handled by the expecta regex, so this would be double-counting?